governance/votes/Executive vote - August 12, 2024.md

```diff
@@ -0,0 +1,184 @@
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### [Stability Fee](https://mips.makerdao.com/mips/details/MIP104#14-3-1-3-stability-fee-sf-) Reductions
+
+- **Authorization**: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-15-sfs-dsr-spark-effective-dai-borrow-rate-reduction/24834/2)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-15-sfs-dsr-spark-effective-dai-borrow-rate-reduction/24834)
+
+If this executive proposal passes, then the following Stability Fee reductions will be made:
+
+- ETH-A: Decrease Stability Fee by 1 percentage point, from 7.25% to **6.25%**.
+- ETH-B: Decrease Stability Fee by 1 percentage point, from 7.75% to **6.75%**.
+- ETH-C: Decrease Stability Fee by 1 percentage point, from 7.00% to **6.00%**.
+- WSTETH-A: Decrease Stability Fee by 1 percentage point, from 8.25% to **7.25%**.
+- WSTETH-B: Decrease Stability Fee by 1 percentage point, from 8.00% to **7.00%**.
+- WBTC-A: Decrease Stability Fee by 1 percentage point, from 8.75% to **7.75%**.
+- WBTC-B: Decrease Stability Fee by 1 percentage point, from 9.25% to **8.25%**.
+- WBTC-C: Decrease Stability Fee by 1 percentage point, from 8.50% to **7.50%**.
+
+### [Dai Savings Rate (DSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-the-dai-savings-rate) Reduction
+
+- **Authorization**: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-15-sfs-dsr-spark-effective-dai-borrow-rate-reduction/24834/2)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-15-sfs-dsr-spark-effective-dai-borrow-rate-reduction/24834)
+
+If this executive proposal passes, then the following reduction to the DSR will be made:
+
+- Decrease the DSR by 1 percentage point, from 7.00% to **6.00%**.
+
+### Bug Bounty Payments
+
+- **Authorization**: [Support Scope 13.1](https://mips.makerdao.com/mips/details/MIP106#13-1-bug-bounty-program-for-makerdao-critical-infrastructure)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/bounty-payout-request-for-immunefi-bug-32005/24605)
+
+If this executive proposal passes, then the following payments as part of the bug bounty program will be distributed.
+
+- **100,000 DAI** will be transferred to [0xA4a6B5f005cBd2eD38f49ac496d86d3528C7a1aa](https://etherscan.io/address/0xA4a6B5f005cBd2eD38f49ac496d86d3528C7a1aa).
+- **10,000 DAI** will be transferred to [0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18 (immunefi.eth)](https://etherscan.io/address/0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18).

```

Comment:
Not sure we want to set the precedent for including ENS names. There's a chance many of payment addresses have one and I'm not sure it adds anything for the voter. Consider dropping.

Reply:
We feel it adds additional information that may aid in the verification process without providing additional overhead. If someone wants to use an ENS we feel that should be supported as long as it is accompanied by the full address as well, within reason.